src/components/kyb/FinancialInformationStep.tsx

```
@@ -0,0 +1,239 @@
+  // Expose validation function to parent component
+  useEffect(() => {
+    // You could emit validation status to parent here if needed
+    const hasErrors = Object.keys(errors).length > 0;
+    const allFieldsFilled = data.annualRevenue && data.primarySourceOfFunds && 
+                           data.expectedMonthlyTransactionVolume && data.businessPurpose;
+    
+    // Store validation status in parent component if needed
+    updateData({ 
+      ...data, 
+      _financialStepValid: !hasErrors && allFieldsFilled 
+    } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)